scripts/composer/post-install.sh

```diff
@@ -1,21 +1,32 @@
 #!/bin/sh
 
+DOCUMENTROOT=web
+
+# Prepare the scaffold files if they are not already present
+if [ ! -f $DOCUMENTROOT/index.php ]
+  then
+    composer drupal-scaffold
+    mkdir -p $DOCUMENTROOT/modules
+    mkdir -p $DOCUMENTROOT/themes
+    mkdir -p $DOCUMENTROOT/profiles
+fi
+
 # Prepare the settings file for installation
-if [ ! -f web/sites/default/settings.php ]
+if [ ! -f $DOCUMENTROOT/sites/default/settings.php ]
   then
-    cp web/sites/default/default.settings.php web/sites/default/settings.php
-    chmod 777 web/sites/default/settings.php
+    cp $DOCUMENTROOT/sites/default/default.settings.php $DOCUMENTROOT/sites/default/settings.php
+    chmod 777 $DOCUMENTROOT/sites/default/settings.php
```

Comment:
Add an echo here (or later below) alerting the user to the chmod? Otherwise, if someone commits without running an install they might accidentally deploy with insecure permissions.